storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/LoggingFactory.java

```diff
@@ -99,7 +99,7 @@ public RequestPolicy create(RequestPolicy next, RequestPolicyOptions options) {
         return new LoggingPolicy(this, next, options);
     }
 
-    private final class LoggingPolicy implements RequestPolicy {
+    private final static class LoggingPolicy implements RequestPolicy {
```

Comment:
what's the reason to make this static?
How is the usage updated after this change?